po/de.po

```diff
@@ -130,8 +130,9 @@ msgstr "Der Ausdruck %s ist falsch."
 
 #: src/string/i18n.c:15
 #, c-format
-msgid "The expression (as strings) %s is false."
-msgstr "Der Ausdruck (als Zeichenkette) %s ist falsch."
+msgid "The expression (as strings) %s is false: actual=`%s` expected=`%s`."
+msgstr ""
+"Der Ausdruck (als Zeichenkette) %s ist falsch: actual=`%s` expected=`%s`."
```

Comment:
I think, we have a slight misunderstanding.
My translation suggestion is
actual -> tatsächlich
expected -> erwartet

btw: http://sprachen-blog.de/false-friends-aktuell-actual/